Tests/SwiftLintFrameworkTests/DisableAllTests.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  DisableAllTests.swift
+//  SwiftLint
+//
+//  Created by Frederick Pietschmann on 3/11/18.
+//  Copyright © 2018 Realm. All rights reserved.
+//
+
+import Foundation
+@testable import SwiftLintFramework
+import XCTest
+
+class DisableAllTests: XCTestCase {
```

Comment:
I also would like to see tests checking the behavior when `(enable|disable):(next|previous|this)` is used, especially nested with other regions.